components/tests/ui/resources/web/tree.txt

```diff
@@ -187,6 +188,13 @@ Select First Dataset With Children
     Wait Until Right Panel Loads    Dataset                  ${datasetId}
     [Return]                        ${datasetId}
 
+Select First Datast With Name
```

Comment:
Have fixed the typo here. And this keyword is being used in the Drag and Drop Negative tests